src/entrypoints/localApp.ts

```diff
@@ -7,14 +7,61 @@ import {
   persistenceModule,
   prepareBuildAndGetDependencies,
 } from '../commands';
+import { Job, Payload } from '../entities/job';
+import { IJobRepoLogger } from '../services/logger';
+
+const fakePayload: Payload = {
+  repoName: 'cloud-docs',
+  project: 'cloud-docs',
+  jobType: '',
+  source: '',
+  action: '',
+  branchName: 'master',
+  isFork: false,
+  repoOwner: '10gen',
+  url: 'https://github.com/10gen/cloud-docs', // ???
+  mutPrefix: 'docs-qa/atlas/mmeigs-build',
+  prefix: '',
+  includeInGlobalSearch: true,
+} as Payload;
+// Bucket??
```

Comment:
Just curious what the question marks are for

Reply:
oh god. I swear I went through multiple times to clean up. This is v old. Deleting.